larray/core/array.py

```diff
@@ -2718,30 +2718,48 @@ def with_total(self, *args, **kwargs):
 
         Examples
         --------
-        >>> arr = ndtest((3, 3))
+        >>> arr = ndtest("age=0..2;time=2013..2016")
```

Comment:
if you want to use something different than the default ndtest (which I am not 100% sure is better in the long run -- **we need to discuss this**), I would rather use sex and time, because having an age between 0 and 2 seems odd to me.